qemu-check.exp

```diff
@@ -72,47 +121,36 @@ send -- "export LD_LIBRARY_PATH=/lib:/lib/arm-linux-gnueabihf\r"
 expect "# "
 info "Running: $cmd...\n"
 send -- "$cmd\r"
-set casenum "none"
-expect {
-	# Exit with error status as soon as a test fails
-	-re {  ([^ ]+) FAIL} {
-		info " $expect_out(1,string) FAIL\n"
-		exit 1
-	}
-	-re {rcu_sched detected stalls} {
-		info " Kernel error: '$expect_out(0,string)'\n"
-		exit 1
-	}
-	# Crude progress indicator: print one # when each test [sub]case starts
-	-re {([\*o]) ([^ ]+) } {
-		set casenum $expect_out(2,string)
-		if {$expect_out(1,string) == "o"} {
-			if {$star == 1} {
-				# Do not count first subcase ('o') since start
-				# of test ('*') was counted already
-				set star 0
-				exp_continue
-			}
-		} else {
-			set star 1
+check_test_result $teecore
+if {$::env(XEN_BOOT) == "y"} {
+	info " Booting DomU.\n"
```

Comment:
Could we have a message to tell we're running Dom0 in the first part of the test above? Something like:
```
info " done, guest is booted"
if {$::env(XEN_BOOT) == "y"} {
        info " (Xen Dom0)"
}
info ".\n"
```

Reply:
Will change